src/test/java/edu/gatech/VRDR/MessagingTest.java

```diff
@@ -910,7 +910,21 @@ public void testCreateSubmissionWithRaceLiterals() {
         assertNotNull(parsed); // make sure we can parse the race values without crashing
     }
 	
-	public void testCreateBulkUploadPayload() {
+    public void testParseBundleOfBundles() {
+        List listOfMessages = BaseMessage.parseBundleOfBundles(DemographicsCodingMessage.class, ctx, "src/test/resources/json/DemographicsCodingMessage.json", null);
```

Comment:
Also test parsing a string without a file, because this is the typical use case.